src/server/server.js

```diff
@@ -169,7 +169,7 @@ Server.prototype._setupSentry = function (configuration) {
     _.extend(sentryOptions, configuration.sentry)
   }
 
-  this.sentry = new Sentry()
+  this.sentry = new Sentry(this.id)
```

Comment:
Yes:
https://github.com/zendesk/radar/blob/edfe8f75780c9fd08553965567f1afa4cf8dd44c/src/core/resources/presence/sentry.js#L30

->

https://github.com/zendesk/radar/blob/edfe8f75780c9fd08553965567f1afa4cf8dd44c/src/core/resources/presence/sentry.js#L109

I think it was adding for testing?